src/store/store.ts

```diff
@@ -73,9 +85,27 @@ const initialState = {
       jobPosition: "Software Developer",
       description:
         "An experienced software engineer with over seven years of experience in the industry. Currently working on a mobile app development project.",
-        skills: [Skills.JavaScript, Skills["C#"], Skills.GoLang, Skills.Java, Skills.Scala],
+      skills: [
+        Skills.JavaScript,
+        Skills["C#"],
+        Skills.GoLang,
+        Skills.Java,
+        Skills.Scala,
+      ],
+    },
+  ],
+  boards: [
+    {
+      name: "Default",
+      columns: [
+        { name: "Todo", statuses: ["new", "committed"] },
+        { name: "In Progress", statuses: ["in progress"] },
+        { name: "Done", statuses: ["done", "fixed"] },
+      ],
+      selected: true,
     },
   ],
+  statuses: ["new", "committed", "in progress", "done", "fixed"],
```

Comment:
I think you can use an enum here just to omit the duplication here. It should not be used in the project as a type. It can be named InitialStatuses.

Reply:
Use lower case char in the enum and don't uppercase it here. Use uppercase in the component or where it is needed.

Reply:
done